packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -254,6 +254,23 @@ async function getOutlookExePath(): Promise<string> {
   }
 }
 
+async function getWXPExePath(app: OfficeApp): Promise<string> {
```

Comment:
This is almost identical to the "getOutlookExePath" method.  I can see the difference is this has been generalized from that which is good, but we should have one method to launch exe files and use that in all cases rather on separate methods for separate cases.

Reply:
You made a good point! Thanks Darren!